view/theme/quattro/dark/style.css

```diff
@@ -469,7 +469,7 @@ blockquote {
   margin-left: 1em;
   border-left: 1em solid #e6e6e6;
 }
-code {
```

Comment:
You forgot tho apply the changes from the LESS file and generate the CSS files from them.

Reply:
How should I do about that again?

Reply:
Edit the less file and run `make` (you need the less compiler installed so that make can complete). In this case it is probably the same change change, so you wont need to recompile the CSS file from the LESS sources.